src/components/Dataset/ContactList.js

```diff
@@ -13,17 +26,44 @@ class ContactList extends React.Component {
     const { contacts } = this.props;
     return (
       <React.Fragment>
+        <Row type="flex" justify="space-between">
+          <h1><FormattedMessage id="datasetContacts" defaultMessage="Dataset contacts"/></h1>
+          <Button htmlType="button" type="primary">
+            <FormattedMessage id="createNew" defaultMessage="Create new"/>
+          </Button>
+        </Row>
+
         <List
           itemLayout="horizontal"
           dataSource={contacts}
           renderItem={item => (
-            <List.Item actions={[<a>edit</a>, <a>more</a>]}>
+            <List.Item actions={[
+              <Button htmlType="button" {...formButton}>
+                <FormattedMessage id="edit" defaultMessage="Edit"/>
+              </Button>,
+              <Button htmlType="button" {...formButton}>
+                <FormattedMessage id="delete" defaultMessage="Delete"/>
+              </Button>
+            ]}>
               <Skeleton title={false} loading={item.loading} active>
                 <List.Item.Meta
-                  title={<Fragment><a
-                    href="">{item.lastName ? `${item.firstName} ${item.lastName}` : item.organization}</a> <span
-                    style={{ fontSize: '12px', color: 'grey', marginLeft: 10 }}>{item.type}</span></Fragment>}
-                  description={<FormattedRelative value={item.created}/>}
+                  title={
+                    <React.Fragment>
+                      {item.lastName ? `${item.firstName} ${item.lastName}` : item.organization}
+                      <span style={{ fontSize: '12px', color: 'grey', marginLeft: 10 }}>
+                        {prettifyUserType(item.type)}
```

Comment:
We would like to be able to translate user types. All enums are translatable please.

Reply:
Sure. Dealing with it right now.